support/Pipelines/Bbh/__init__.py

```diff
@@ -19,6 +20,10 @@ def list_commands(self, ctx):
         ]
 
     def get_command(self, ctx, name):
+        if name == "ecc-control":
+            from .EccentricityControl import ecc_control_fit
```

Comment:
This will need to import `eccentricity_control_command` (which doesn't exist yet)

Reply:
do in next PR